src/providers/twitch/TwitchChannel.cpp

```diff
@@ -1246,4 +1298,12 @@ boost::optional<CheerEmote> TwitchChannel::cheerEmote(const QString &string)
     return boost::none;
 }
 
+void TwitchChannel::listenSeventv()
+{
+    if (getApp()->twitch->eventApi)
```

Comment:
If you wanna be consistent with other changes you've made, you could do this to avoid another call to getApp and avoid repetition
```cpp
if (auto eventApi = getApp()->twitch->eventApi)
        eventApi->joinChannel(this->getName());
```
but that's honestly just nitpicking